### DISCUSSION TITLE
[Request] Live move for all motors

### DISCUSSION THREAD

**Discussion by Giadej** [Jyers/Marlin#466] [Ideas]
Hi, it is possible to implement the "Live move for all motors" function (except for extruder) in order to move the axes without having to first assign the relative value from the lcd.
Thank you in advance

**Comment by Jyers** [comment]
I'm not entirely sure what you're asking for. How would this differ from the move menu?

**Reply by Giadej** [reply]
Hi Jyer, this is a function that allows you to move the motors (once the axis XYZ has been selected on lcd) simply by turning the knob.

**Reply by Jyers** [reply]
Why is the advantage of this compared to the current method?

**Reply by Giadej** [reply]
Because with the current metod you must select the axes, insert the value, click the knob and attempt to movement finish to insert the new value. With the "live move motor" you can, for example, raise and lower the X axes in real time for clean the nozzle without insert the values. This function already exists on another version of E3v2 firmware "custom".

**Reply by joeclifford** [reply]
I knew I'd seen a pull upstream for this.

https://github.com/MarlinFirmware/Marlin/pull/21035

Looks like this is now the default behaviour for marlin.

**Reply by Jyers** [reply]
I'll look into implementing this

**Reply by tome9111991** [reply]
I dont like the live move in stock marlin.
Maybe u can add a checkbox to enable/disable live movement?

**Reply by Jyers** [reply]
That's what I was planning on doing

**Reply by DVCasey0311** [reply]
I know this has been implemented already but it works erratically for me, when I use this to move the z axis up it will move up and then back down then back up and then settle at the starting z position. its very weird.
